### DISCUSSION TITLE
point collection system

### DISCUSSION THREAD

**Discussion by skywalk08189557** [IBAX-io/go-ibax#12] [Typos （English）]
I joined Discord, telegram groups. I invited my friend. My friend registered via the invitation link I sent, but the points were not defined in my account. I only get points for retweet posts. I think you can make this system healthier. Thank you.

**Comment by HalloKyle** [comment]
Completing the retweet task will earn you IBAX points. But If you invite a friend, you'll get IBXC coins straight away!

**Reply by skywalk08189557** [reply]
I made my friend. Joined through my referral code and fulfilled the tasks. But I didn't get any points

**Reply by HalloKyle** [reply]
You will receive IBXC coins directly! Please check the rules, IBAX points are used to redeem IBXC coins.

**Reply by bekirdamar001** [reply]
How can I win ibax point

**Reply by HalloKyle** [reply]
Join the airdrop event
https://www.ibax.io/login

**Reply by Yenen21** [reply]
İbax puanları olmadan ıbxc paralarını kulanamıyormuyuz

**Comment by Sancaklimehmet** [comment]
Retweet görevlerini yaptım 8500 puan aldım.Telegramdan puan alamadım.Burdan nasıl  puan alacam onuda bilmiyorum  açıkçası

**Reply by Sancaklimehmet** [reply]
Bu işi nasıl çözeceğim bilmiyorum.

**Reply by Sancaklimehmet** [reply]
Hanı bana puan yokmu:)
